### PR TITLE
#160521468 Improve code maintainability

### PR DESCRIPTION
**Pull request by davidshare**
#### What does this PR do?
Improve the code maintainability

#### Description of Task to be completed?
- remove rules for checking validity of user input from the ValidateOrder class
- create a separate module to hold rules for validation
- remove strings for error message from ValidateOrder class
- create a separate module for error messages
- fix grammatical error in message for invalid orderId in the orders test file

#### How should this be manually tested?
- visit the repository on GitHub and check the readme file for the maintainability badge

#### What are the relevant pivotal tracker stories?
#160521468

### CODE REVIEW

File: server/middleware/orderValidator.js [hunk 1]
```diff
@@ -26,132 +29,156 @@ class ValidateOrder {
...
       });
     }
     return next();
   }
 
-  static validateOrderId(request, response, next) {
+  /**
+   * check validation errors
+   * @param {Object} request
+   *
```
Comment (KaiserPhemi) [inline server/middleware/orderValidator.js]: #### Nitpick
This space is unnecessary

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

**Comment by davidshare** [comment]
Sorry I don't really understand. Should I remove the empty lines in the comment block or the entire comment.

File: server/middleware/orderValidator.js (hunk 1 above)
Comment (davidshare) [reply server/middleware/orderValidator.js]: Sorry, I don't really understand. Should I remove the empty lines in the comment block or the entire comment?

Comment (KaiserPhemi) [reply server/middleware/orderValidator.js]: @davidshare 
Just the empty line

**Review by KaiserPhemi — APPROVED** [review]

**Comment by KaiserPhemi** [comment]
Looks Good To Me (LGTM) but try & increase the Code Coverage.

**Pull request merged**